pkg/cli/admin/upgrade/status/workerpool.go

```diff
@@ -472,24 +471,25 @@ func machineConfigPoolInsights(poolDisplay poolDisplayData, pool mcfgv1.MachineC
 	return insights
 }
 
-var workerPoolStatusTemplate = template.Must(template.New("workerPoolStatus").Parse(workerPoolStatusTemplateRaw))
-
-func (pool *poolDisplayData) WritePool(f io.Writer) error {
-	return workerPoolStatusTemplate.Execute(f, pool)
+func writePools(w io.Writer, workerPoolsStatusData []poolDisplayData) {
+	tabw := tabwriter.NewWriter(w, 0, 0, 3, ' ', 0)
+	_, _ = tabw.Write([]byte("\nWorker Pool\tAssessment\tCompletion\tStatus\n"))
+	for _, pool := range workerPoolsStatusData {
+		_, _ = tabw.Write([]byte(pool.Name + "\t"))
+		_, _ = tabw.Write([]byte(pool.Assessment + "\t"))
+		_, _ = tabw.Write([]byte(fmt.Sprintf("%.0f%%", pool.Completion) + "\t"))
+		_, _ = tabw.Write([]byte(fmt.Sprintf("%d Total, %d Available, %d Progressing, %d Outdated, %d Draining, %d Excluded, %d Degraded",
+			pool.NodesOverview.Total, pool.NodesOverview.Available, pool.NodesOverview.Progressing, pool.NodesOverview.Outdated,
+			pool.NodesOverview.Draining, pool.NodesOverview.Excluded, pool.NodesOverview.Degraded) + "\n"))
+	}
+	tabw.Flush()
 }
 
-const workerPoolStatusTemplateRaw = `= Worker Pool =
-Worker Pool:     {{ .Name }}
-Assessment:      {{ .Assessment }}
-Completion:      {{ printf "%.0f" .Completion }}%
-Worker Status:   {{ .NodesOverview.Total }} Total, {{ .NodesOverview.Available }} Available, {{ .NodesOverview.Progressing }} Progressing, {{ .NodesOverview.Outdated }} Outdated, {{ .NodesOverview.Draining }} Draining, {{ .NodesOverview.Excluded }} Excluded, {{ .NodesOverview.Degraded }} Degraded
-`
-
 func (pool *poolDisplayData) WriteNodes(w io.Writer, detailed bool) {
 	if pool.Name == mco.MachineConfigPoolMaster {
 		fmt.Fprintf(w, "\nControl Plane Node")
 	} else {
-		fmt.Fprintf(w, "\nWorker Pool Node")
+		fmt.Fprintf(w, "\nWorker Pool Node: %s", pool.Name)
```

Comment:
```suggestion
		fmt.Fprintf(w, "\nWorker Pool Nodes: %s", pool.Name)
```

Now it does not make sense to try being smart about the plural (L494-496), let's just make the title `Worker Pool Nodes` always